### ISSUE TITLE
Show a tangram/view link after a Gist is saved.

### ISSUE THREAD

**Issue by burritojustice** [tangrams/tangram-play#481]
I'd find it very helpful if we could add a tangram-frame URL in the .tangramplay gist blob. Right now I usually go in, copy the link to the raw yaml file, drop the commit number:

https://gist.githubusercontent.com/burritojustice/f4aae82a0bd4a0cd985a3e263d5a9057/raw/scene.yaml

(Note if we drop the commit number (the second number in the raw URL), any edits made in the gist will be automatically updated in tangram-frame.)

I then paste the shortened URL to the end of the tangram-frame URL and add out a lat/lon and zoom level.

http://tangrams.github.io/tangram-frame/?url=https://gist.githubusercontent.com/burritojustice/f4aae82a0bd4a0cd985a3e263d5a9057/raw/scene.yaml#7.22923/56.193/11.623

This is error prone and time consuming.

Ideally we could append the lat/lon and zoom from the other metadata from the view we already save in .tangramplay.

A (much shorter) alternative would be to append just the gist ID to tangram-frame and teach to look for the yaml, coordinates and zoom in the API response.

http://tangrams.github.io/tangram-frame/?gist=f4aae82a0bd4a0cd985a3e263d5a9057

gist api response, for reference:

https://api.github.com/gists/f4aae82a0bd4a0cd985a3e263d5a9057

**Comment by louh** [comment]
Related conversation: @meetar, what do you think about mounting tangram-frame to https://mapzen.com/tangram/view/?

**Comment by meetar** [comment]
Fine with me!

**Comment by louh** [comment]
Cool. @meetar, when you get back let's go over tangram-frame and see what we would need to do to make it production-url ready. Then we can have canonical view-only URLs for scenes!

**Comment by louh** [comment]
I've just realized we cannot save a link to the gist metadata. To understand why, imagine this sequence of events:

1. I have a scene I want to save. I package everything up and send it to GitHub.
2. GitHub acknowledges the request. GitHub saves my scene to their server, then responds with the URL information of where they saved it to.
3. I can now create a link! ... but... I have already saved the metadata!

Since it is impossible to know what the link is _before_ I save it, we can't actually do it this way.

We can still expose the URL to the view in the copy/paste dialog that occurs after confirmation. This will be the current goal for this issue.
